detekt.yml

```diff
@@ -60,6 +60,6 @@ empty-blocks:
 
 performance:
     ForEachOnRange:
```

Comment:
This rule could be removed completely from the detekt.yml.

Reply:
Done